spec/index.ts

```diff
@@ -4,10 +4,10 @@ import dateConverter from './common/dateconverter'
 
 class Student {
     @JsonProperty('name')
-    fullName: string;
+    fullName?: string;
```

Comment:
As shown below, it's assigned with undefined, which could not pass the strict null checking:

![image](https://cloud.githubusercontent.com/assets/2306105/22676558/1828ca5c-ed27-11e6-994d-753a873a6cea.png)